corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -248,7 +248,7 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
         let isRequired = self.question.required() ? "Yes" : "No";
         $(function () {
             let entry = $(`#${self.entryId}`);
-            entry.change(function () {
```

Comment:
This is deprecated? I don't see it mentioned in the docs: https://api.jquery.com/change/